### ISSUE TITLE
Rendering Techniques: <Add a better props explanation>

### ISSUE THREAD

**Issue by mhamza-khalid** [TheOdinProject/curriculum#26901]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

I am currently studying the React section of TOP and my journey so far has been amazing with TOP! 
However in the lesson 'Rendering Techniques' there could be some improvements so the content could be more easy to digest. In the '[Rendering a list of components in JSX]' part it explains very little what props is and the code that follows is hard to understand without a fundamental understanding of how props work. I had to go to the React documentation (https://react.dev/learn/passing-props-to-a-component) and after reading this I was able to understand how the code in [Rendering a list of components in JSX] works. My suggestion is to please include a link of the props lesson of React before starting '[Rendering a list of components in JSX]'. Thank you!

### Path

Other / NA

### Lesson Url

https://www.theodinproject.com/lessons/node-path-react-new-rendering-techniques

**Comment by JoshDevHub** [comment]
Thoughts on this? @TheOdinProject/javascript

**Comment by 01zulfi** [comment]
@mhamza-khalid, thanks for creating an issue.

Not explaining how props work is intentional. We use props in the examples to show "conditionally rendering UI". This is practical because props often determine what to render in React components We even have a note in the lesson on the use of props and that we will deep dive on them in a later lesson. As such, I don't think we need to add the mentioned resource as that already is present in the [prop lesson's assignment section](https://www.theodinproject.com/lessons/node-path-react-new-passing-data-between-components#assignment).

What do you think?

**Comment by mhamza-khalid** [comment]
Yes as I moved on the course I understood that why it was intentional. Thank you for your time and guidance.
Have a great weekend ahead @01zulfi!

**Comment by mhamza-khalid** [comment]
Yes as I moved on the course I understood that why it was intentional. Thank you for your time and guidance.
Have a great weekend ahead @01zulfi!
